tests/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/onflow/flow-evm-gateway/integration
 
-go 1.20
+go 1.21
+
+toolchain go1.21.4
```

Comment:
Locally I get this error:
```bash
go: errors parsing go.mod:
/home/mpeter/Dev/flow-evm-gateway/tests/go.mod:5: unknown directive: toolchain
```

Reply:
hmm not sure how that got in, thanks for flagging